diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/VariableGenericTypeDeclarationRule.kt

```diff
@@ -33,11 +35,14 @@ class VariableGenericTypeDeclarationRule(private val configRules: List<RulesConf
 
     @Suppress("UnsafeCallOnNullableType")
     private fun handleProperty(node: ASTNode) {
+        val callExpr = if (node.findChildByType(CALL_EXPRESSION) != null) {
+            node.findChildByType(CALL_EXPRESSION)
+        } else {
+            node.findChildByType(DOT_QUALIFIED_EXPRESSION)?.getAllChildrenWithType(CALL_EXPRESSION)?.lastOrNull()
+        }
```

Comment:
```suggestion
        val callExpr = if (node.hasChildOfType(CALL_EXPRESSION)) {
            node.findChildByType(CALL_EXPRESSION)
        } else {
            node.findChildByType(DOT_QUALIFIED_EXPRESSION)?.getAllChildrenWithType(CALL_EXPRESSION)?.lastOrNull()
        }
```
or you can just use `?:`